plugins/chain_plugin/trx_retry_db.cpp

```diff
@@ -52,11 +52,14 @@ struct tracked_transaction {
    }
 
    size_t memory_size()const { return ptrx->get_estimated_size() + trx_trace_v.estimated_size() + sizeof(*this); }
-
+#if __cplusplus >= 202002L
+   [[no_unique_address]] noncopyable _;
+#else
    tracked_transaction(const tracked_transaction&) = delete;
    tracked_transaction() = delete;
    tracked_transaction& operator=(const tracked_transaction&) = delete;
    tracked_transaction(tracked_transaction&&) = default;
+#endif
```

Comment:
I think we should instead just add the needed constructor that will work in pre-c++20 as well.

Reply:
Does it work though? We are using the aggregate initialization when inserting into the boost multi-index, and adding any declared constructor makes the struct not an aggregate anymore in c++20.

Reply:
Anything is better than than #ifdef. Find a solution that works for both. Easiest is probably to just remove the `=delete`. This is all in a cpp file, we don't need those.

Reply:
I'll remove the `#ifdef` once we switch the build to `-std=c++20`. This is just temporary.

Reply:
I think this is rather ugly, would not want to see the addition of `struct noncopyable` and `#ifdef` even temporarily.

Reply:
OK, I removed the whole `#ifdef`.